server/api.js

```diff
@@ -211,7 +165,34 @@ router.post("/laptop_donation", (req, res) => {
 		email,
 		deliveryOption,
 	])
-		.then(() => res.send("result.rows"))
+		.then(async () => {
+			let laptopDonationResult = await db.query(
```

Comment:
the mixture of then and await is a mess. Use one or the other (maybe in a followup pull request though)

Reply:
I think it is necessary to use both, I could demo why I think that